templates/rango/about.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+{% load staticfiles %}
+
+<html>
+
+    <head>
+        <title>Rango</title>
+    </head>
+
+    <body>
+        <h1>Rango says...</h1>
+
+        <div> here is the about page. </div>
+
+        <div>
+            hey there partner! <br />
+            <strong>{{ message }}</strong><br />
+        </div>
+
+        <div>
+            <a href="/rango/">Index</a><br />
+            <img src="{% static 'images/rango.jpeg' %}"
+                 alt="Picture of Rango" />
+            <img src="{{ MEDIA_URL }}sexy_woman.jpeg" alt="Picture of a Hot Baby."
```

Comment:
why do you use 2 different ways to put the image?
1. {% static 'images/rango.jpeg' %}
2. {{ MEDIA_URL }}sexy_woman.jpeg

it is better to use the first option.